packages/experimental/RadioGroup/src/Radio/Radio.types.ts

```diff
@@ -0,0 +1,96 @@
+import type { IViewProps } from '@fluentui-react-native/adapters';
+import { TextProps } from '@fluentui-react-native/text';
+import { FontTokens, IForegroundColorTokens, IBackgroundColorTokens, IBorderTokens, IColorTokens } from '@fluentui-react-native/tokens';
+import { IPressableProps } from '@fluentui-react-native/pressable';
+import { IFocusable, IPressableHooks } from '@fluentui-react-native/interactive-hooks';
+import { ColorValue } from 'react-native';
+
+export const radioName = 'Radio';
+
+export interface RadioTokens extends FontTokens, IColorTokens, IForegroundColorTokens, IBackgroundColorTokens, IBorderTokens {
+  /**
+   * Focus border color
+   */
+  // textBorderColor?: ColorValue;
+
+  /**
+   * Indicator border color
+   */
+  radioBorder?: ColorValue;
+
+  /**
+   * Inner circle color when selected
+   */
+  radioFill?: ColorValue;
+
+  /**
+   * Label text color
+   */
+  // labelColor?: ColorValue;
```

Comment:
Is the commented out token going to be used later? If not, commented code should be deleted/not checked in

Reply:
Just didn't get around to cleaning up the code yet! Removed now.